parse_mem.py

```diff
@@ -1,151 +1,90 @@
-#!/usr/bin/python
```

Comment:
There's a recommended shebang for executable Python 3 files per [PEP 394](https://www.python.org/dev/peps/pep-0394/#recommendation):

```suggestion
#!/usr/bin/env python3
```